presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -195,6 +203,144 @@ public void testClusteringKeyOnlyPushdown()
         assertEquals(execute(sql).getRowCount(), 1);
     }
 
+    @Test
+    public void testUppercaseNameUnescaped()
```

Comment:
Maybe testUpperCaseNameUnescapedInCassandra? The comment right below this makes it confusing what you're doing.